scripts/kind-e2e/e2e.sh

```diff
@@ -36,9 +37,9 @@ function test_with_e2e_tests {
 declare_kubeconfig
 PRJ_ROOT=$(git rev-parse --show-toplevel)
 
-run_parallel "2 3" update_coredns_configmap
+run_subm_clusters update_coredns_configmap
 import_image lighthouse-coredns
-run_parallel "{1..3}" update_coredns_deployment
+run_subm_clusters update_coredns_deployment
```

Comment:
Interesting that the `run_subm_clusters`  calls failed as expected in the CI jobs but the jobs still succeeded 😄

Reply:
Yes it's because we dont `set -o pipefail` on all the scripts so some places its not set and then even if it fails the E2E the script doesn't exit so the job thinks all is good.. :/